mvc/views/tag-helpers/authoring.rst

```diff
@@ -1,8 +1,376 @@
-.. include:: /../common/stub-topic.txt
-
-|stub-icon| Authoring Tag Helpers
+Authoring Tag Helpers
 ===========================================
 
-.. include:: /../common/stub-notice.txt
+By `Rick Anderson`_
+
+    - `Getting started with Tag Helpers`_
```

Comment:
Tag Helpers when referred to as a feature is a proper noun. It is a particular person/place/thing. Ex "One of the great new features in MVC 6 is Tag Helpers."

If you are talking about tag helpers in general then I think lc is fine. Ex "To update the content of a tag helper you need to  . . ."